config.go

```diff
@@ -476,6 +515,30 @@ func getConfig() *types.Configuration {
 	v.SetDefault("Dynatrace.CheckCert", true)
 	v.SetDefault("Dynatrace.MinimumPriority", "")
 
+	v.SetDefault("OTLP.Traces.Endpoint", altEnvs([]string{
+		"OTEL_EXPORTER_OTLP_TRACES_ENDPOINT",
+		"OTEL_EXPORTER_OTLP_ENDPOINT+/v1/traces",
+	}))
+	v.SetDefault("OTLP.Traces.Protocol", altEnvs([]string{
+		"OTEL_EXPORTER_OTLP_TRACES_PROTOCOL",
+		"OTEL_EXPORTER_OTLP_PROTOCOL",
+	}))
+	v.SetDefault("OTLP.Traces.Headers", altEnvs([]string{
+		"OTEL_EXPORTER_OTLP_TRACES_HEADERS",
+		"OTEL_EXPORTER_OTLP_HEADERS",
+	}))
+	v.SetDefault("OTLP.Traces.Timeout", altEnvs([]string{
+		"OTEL_EXPORTER_OTLP_TRACES_TIMEOUT",
+		"OTEL_EXPORTER_OTLP_TIMEOUT",
+	}))
```

Comment:
I understand these envs vars are the default ones used by the sdk, but I prefer to expose them in the settings as specific settings. It seems to repeat things but it will clearer for users and also easier for the implementation in the helm chart later.

Moreover, for `OTLP.Traces.Timeout` it can't work, because it's an `int64` and `altEnvs()` returns string.

I understand what you're trying to do, but it will introduce too much complexity when setting all explicitly could work.

Reply:
```suggestion
	v.SetDefault("OTLP.Traces.Endpoint", "")
	v.SetDefault("OTLP.Traces.Protocol", "http")
	v.SetDefault("OTLP.Traces.Headers", "")
	v.SetDefault("OTLP.Traces.Timeout", 1000)
```

Reply:
> why are you using a slice of string and not a map? i think the iteration over a slice is not consistent, the order is not always kept, it is in a map

(?) I think is the other way round, as a slice is essentially a header that specifies a range for an underlying array, from https://go.dev/ref/spec#For_statements ->

1. For an array, pointer to array, or slice value a, the **index iteration values are produced in increasing order**, starting at element index 0. If at most one iteration variable is present, the range loop produces iteration values from 0 up to len(a)-1 and does not index into the array or slice itself. For a nil slice, the number of iterations is 0.
2. [...]
3. The **iteration order over maps is not specified and is not guaranteed to be the same from one iteration to the next**. If a map entry that has not yet been reached is removed during iteration, the corresponding iteration value will not be produced. If a map entry is created during iteration, that entry may be produced during the iteration or may be skipped. The choice may vary for each entry created and from one iteration to the next. If the map is nil, the number of iterations is 0.

Reply:
> I understand these envs vars are the default ones used by the sdk, but I prefer to expose them in the settings as specific settings. It seems to repeat things but it will clearer for users and also easier for the implementation in the helm chart later.
> 

I'd rather advocate for this one: I do want to keep `OTEL_*` envvars compatibility, to let people already working with OTEL and familiar with those env vars, to let the sidekick behave as any other OTEL application (mind that the OTEL envvars spec is cross-implementation also) -- can think of known OTEL env vars already being set from Kubernetes secrets for example.

> Moreover, for `OTLP.Traces.Timeout` it can't work, because it's an `int64` and `altEnvs()` returns string.

+1 can fix it of course (if you agree to keeping this OTEL env vars handling)

> 
> I understand what you're trying to do, but it will introduce too much complexity when setting all explicitly could work.

🤷🏼 I know, but IMO the tradeoff to let this behave as any other OTEL app weights more =)

Reply:
> Moreover, for OTLP.Traces.Timeout it can't work, because it's an int64 and altEnvs() returns string.

BTW this does work, note the  `OTEL_EXPORTER_OTLP_TRACES_TIMEOUT=4242` env being set ->
```
$ go build -o falcosidekick
$ OTLP_TRACES_SYNCED=true \
  OTLP_TRACES_CHECKCERT=false \
  OTLP_TRACES_ENDPOINT=http://172.17.0.1:4318/v1/traces \
  OTEL_EXPORTER_OTLP_TRACES_TIMEOUT=4242 ./falcosidekick
2023/09/06 18:59:01 [INFO] : OTLP.Traces={Endpoint:http://172.17.0.1:4318/v1/traces Protocol: Timeout:4242 Headers: Duration:1000 CheckCert:false Synced:true MinimumPriority: TraceIDHash: TraceIDHashTemplate:<nil>}
2023/09/06 18:59:01 [INFO]  : Falco Sidekick version: devel
2023/09/06 18:59:01 [INFO]  : Enabled Outputs : [OTLPTraces]
2023/09/06 18:59:01 [INFO]  : Falco Sidekick is up and listening on :2801
```